test case `test_025_packet_count_per_flow` is a bit flaky

Over the past months, `tests/test_e2e_70_kytos_stats.py::TestE2EKytosStats::test_025_packet_count_per_flow` has failed 6 times, although reruns managed to fix it, so we need to improve this test case to be more reliable. Maybe stats got a bit delayed with high cpu usage and stats didn't arrive in time, it might need an extra sleep:

```
tests/test_e2e_70_kytos_stats.py::TestE2EKytosStats::test_025_packet_count_per_flow: 2024-09-21,07:49:17.888929 - 2024-09-21,07:49:17.932513
self = <tests.test_e2e_70_kytos_stats.TestE2EKytosStats object at 0x7fecefe59350>

    def test_025_packet_count_per_flow(self):
        """Test packet_count_per_flow"""
        api_url = KYTOS_STATS + '/flow/stats?dpid=00:00:00:00:00:00:00:01'
        response = requests.get(api_url)
        assert response.status_code == 200, response.text
        data_flow = response.json()['00:00:00:00:00:00:00:01']

        api_url = KYTOS_STATS + '/packet_count/per_flow/00:00:00:00:00:00:00:01'
        response = requests.get(api_url)
        assert response.status_code == 200, response.text
        data = response.json()
        for info_flow in data:
            flow_id = info_flow['flow_id']
            count = data_flow[flow_id]['packet_count']
            assert info_flow['packet_counter'] == count
>           assert info_flow['packet_per_second'] == count/data_flow[flow_id]['duration_sec']
E           ZeroDivisionError: division by zero

tests/test_e2e_70_kytos_stats.py:233: ZeroDivisionError
=========================== rerun test summary info ============================
RERUN tests/test_e2e_70_kytos_stats.py::TestE2EKytosStats::test_025_packet_count_per_flow
= 243 passed, 8 skipped, 9 xfailed, 7 xpassed, 1295 warnings, 1 rerun in 13610.86s (3:46:50) =
```


```
rerun: 0
tests/test_e2e_70_kytos_stats.py::TestE2EKytosStats::test_025_packet_count_per_flow: 2024-09-24,04:02:19.285263 - 2024-09-24,04:02:19.319901
self = <tests.test_e2e_70_kytos_stats.TestE2EKytosStats object at 0x7fb52d326ad0>

    def test_025_packet_count_per_flow(self):
        """Test packet_count_per_flow"""
        api_url = KYTOS_STATS + '/flow/stats?dpid=00:00:00:00:00:00:00:01'
        response = requests.get(api_url)
        assert response.status_code == 200, response.text
        data_flow = response.json()['00:00:00:00:00:00:00:01']

        api_url = KYTOS_STATS + '/packet_count/per_flow/00:00:00:00:00:00:00:01'
        response = requests.get(api_url)
        assert response.status_code == 200, response.text
        data = response.json()
        for info_flow in data:
            flow_id = info_flow['flow_id']
            count = data_flow[flow_id]['packet_count']
            assert info_flow['packet_counter'] == count
>           assert info_flow['packet_per_second'] == count/data_flow[flow_id]['duration_sec']
E           ZeroDivisionError: division by zero
```

--- a/tests/test_e2e_70_kytos_stats.py
+++ b/tests/test_e2e_70_kytos_stats.py
@@ -217,6 +217,9 @@ class TestE2EKytosStats:
 
     def test_025_packet_count_per_flow(self):
         """Test packet_count_per_flow""" 
+        # give enough time for stats gathering (of_core.STATS_INTERVAL)
+        time.sleep(10)
+
         api_url = KYTOS_STATS + '/flow/stats?dpid=00:00:00:00:00:00:00:01'  
         response = requests.get(api_url)
         assert response.status_code == 200, response.text
@@ -229,12 +232,14 @@ class TestE2EKytosStats:
         for info_flow in data:
             flow_id = info_flow['flow_id']
             count = data_flow[flow_id]['packet_count']
-            assert info_flow['packet_counter'] == count
-            assert info_flow['packet_per_second'] == count/data_flow[flow_id]['duration_sec']
+            assert info_flow['packet_counter'] >= count
+            assert info_flow['packet_per_second'] >= count/data_flow[flow_id]['duration_sec']
 
 
     def test_030_bytes_count_per_flow(self):
         """Test bytes_count_per_flow""" 
+        # give enough time for stats gathering (of_core.STATS_INTERVAL)
+        time.sleep(10)
 
         api_url = KYTOS_STATS + '/flow/stats?dpid=00:00:00:00:00:00:00:01'  
         response = requests.get(api_url)
@@ -248,8 +253,8 @@ class TestE2EKytosStats:
         for info_flow in data:
             flow_id = info_flow['flow_id']
             count = data_flow[flow_id]['byte_count']
-            assert info_flow['bytes_counter'] == count
-            assert info_flow['bits_per_second'] == 8 * count/data_flow[flow_id]['duration_sec']
+            assert info_flow['bytes_counter'] >= count
+            assert info_flow['bits_per_second'] >= 8 * count/data_flow[flow_id]['duration_sec']
 
     def test_035_table_fields_update(self):
         """Test fields are updating on table 0.

